.storybook/stories/datagrid/datagrid-detail.stories.ts

```diff
@@ -154,3 +156,11 @@ const DetailTemplate: StoryFn = args => {
 export const Detail: StoryObj = {
   render: DetailTemplate,
 };
+
+export const OpenDetail: StoryObj = {
+  render: DetailTemplate,
+  args: {
+    opened: elements[4],
+    height: 500,
```

Comment:
```suggestion
    // The height is set larger than the height of the rows to regression test the detail pane border. (CDE-2188)
    height: 500,
```

I think it would be good to have a comment here explaining the height.